### DISCUSSION TITLE
Bots should load Railgun/Coilgun Loader with ammo if there are slots available when Cleaning Up.

### DISCUSSION THREAD

**Discussion by Kaillera** [FakeFishGames/Barotrauma#4758] [Ideas and suggestions]
Bots should load Railgun/Coilgun Loader with ammo if there are slots available when Cleaning Up.

Current Workaround: I found that tagging the loaders in our own subs to mimic racks allows bots to do this, and it's very beneficial to have.

**Comment by Sviat** [comment]
There is property in Sub-Editor: Container Restrictions, you can put there railgunammo , and bots start doing this.

Would be cool if we can do something like this in-game.
